README.md

```diff
@@ -44,15 +44,23 @@ script manually.
 <summary><h4>Required tower upgrades</h4></summary>
 
 - Dart Monkey 024
-- Sniper Monkey 322
+- Sniper Monkey 425
 - Monkey Sub 223
 - Monkey Buccaneer 032
+- Monkey Ace 205
 - Wizard Monkey 032
+- Alchemist 420
 - Druid 130
+- Banana Farm 424
 - Spike Factory 003
+- Monkey Village 234
+- Benjamin
```

Comment:
These should probably be separated by difficulty but it's nice that you updated this.